packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -14,7 +14,7 @@ import 'term.dart';
 enum _SymbolType { invalid, external, slack, error, dummy, }
 
 class _Symbol {
-  const _Symbol(this.type);
```

Comment:
Trying to use `const _Symbol(...)` instead of  `new _Symbol(...)` causes several errors in tests. Actually there's no  `const _Symbol(...)` in the code base (more accurately `solver.dart` as it's a private class).
Those thing made me think that `_Symbol` was not really intended to be used as `const` and that's why I removed the `const` here.

wdyt ?